administrator/components/com_joomlaupdate/src/Model/NotificationModel.php

```diff
@@ -45,14 +45,26 @@ final class NotificationModel extends BaseDatabaseModel
      * @since   5.4.0
      */
     public function sendNotification($type, $oldVersion): void
-    {
+    { 
```

Comment:
```suggestion
    {
```
Remove space at the end of line.